spec/factories/omniauth.rb

```diff
@@ -0,0 +1,33 @@
+FactoryGirl.define do
+
+  factory :auth_hash, class: OmniAuth::AuthHash do
+    skip_create
+
+    initialize_with do
+      OmniAuth::AuthHash.new({
+        provider: provider,
+        uid: uid,
+        info: {
+          email: email
+        }
+      })
```

Comment:
This factory will now generate exceptions if it's called without providing a trait because `provider`, `uid`, and `email` will be undefined. That seems too brittle, so could you please add some non-transient, default values? I'm sorry if my comment about transient attributes earlier introduced any confusion.